packages/flutter/lib/src/widgets/feedback.dart

```diff
@@ -141,6 +142,9 @@ abstract final class Feedback {
       case TargetPlatform.fuchsia:
         return HapticFeedback.vibrate();
       case TargetPlatform.iOS:
+        SystemSound.play(SystemSoundType.click);
+        HapticFeedback.heavyImpact();
+        return Future<void>.value();
```

Comment:
Nit: Use [Future.wait](https://main-api.flutter.dev/flutter/dart-async/Future/wait.html) to return a Future that will resolve when both of these Futures have finished.